modules/ocf_kubernetes/manifests/ingress.pp

```diff
@@ -0,0 +1,32 @@
+class ocf_kubernetes::ingress {
+
+  $kubernetes_workers_ipv4 = lookup('kubernetes::workers_ipv4')
+
+  file {
+    default:
+      require =>  Package['kubeadm', 'kubectl'];
+    '/etc/kubernetes/manifests/ingress':
+      ensure => directory,
+      mode   => '0700';
+    '/etc/kubernetes/manifests/ingress/ingress-expose.yaml':
+      content => template('ocf_kubernetes/ingress/ingress_expose.yaml.erb'),
+      mode    => '0644';
+  }
+
+  # Add ingress-nginx to the cluster
+  exec { 'init-ingress':
+    command => 'kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/mandatory.yaml',
```

Comment:
You will want to make sure this is only executed once

Reply:
Kubernetes will ignore it if the manifest doesn't change.

Reply:
Still I think we should pick a tag instead of running directly from their master branch